xarray/core/ops.py

```diff
@@ -52,7 +52,9 @@
 NAN_CUM_METHODS = ['cumsum', 'cumprod']
 BOTTLENECK_ROLLING_METHODS = {'move_sum': 'sum', 'move_mean': 'mean',
                               'move_std': 'std', 'move_min': 'min',
-                              'move_max': 'max'}
+                              'move_max': 'max', 'move_var': 'var',
+                              'move_argmin': 'argmin', 'move_argmax': 'argmax',
+                              'move_rank': 'rank'}
```

Comment:
I don't think we have a `rank` method `DataArray`/`Dataset`, so we should probably remove this for now for consistency (though it would be nice to add it later!). Also I think it's untested.